code/game/say.dm

```diff
@@ -126,6 +126,15 @@ var/list/freqtoname = list(
 		pooled=1
 	else
 		say_testing(speech.speaker," We <i>do</i> understand this gentle\[wo\]man.")
+		//checking for syndie codephrases if person is a tator
+		if(istype(src, /mob/living))
+			var/mob/M = src
+			if(M.mind.special_role == TRAITOR || M.mind.special_role == NUKE_OP)
```

Comment:
`special_role` is a relic of old times
Use `M.mind.GetRole(TRAITOR)` or `M.mind.GetRole(NUKE_OP)`

Reply:
done.